apps/aechannel/test/aesc_utils_tests.erl

```diff
@@ -262,8 +262,8 @@ deserialize_broken_close_solo_payload() ->
     {error, bad_offchain_state_type} =
         aesc_utils:check_solo_close_payload(ChannelPubKey,
                                             FromPubKey,
-                                            _Nonce = 1,
-                                            _Fee = 1,
+                                            _Nonce1 = 1,
```

Comment:
Wouldn't it be nice to pass the parameters as map elements instead? (could be a separate PR)